config/rails_best_practices.yml

```diff
@@ -28,7 +28,8 @@ RemoveUnusedMethodsInHelpersCheck: { except_methods: [
   'GithubEventPayloadHelper#occurence_time',
```

Comment:
Can you remove this and any other method exceptions that are no longer valid?

Reply:
Yep!